lib/matplotlib/tight_layout.py

```diff
@@ -172,15 +172,15 @@ def auto_adjust_subplotpars(
     if margin_left + margin_right >= 1:
         margin_left = 0.4999
         margin_right = 0.4999
-        matplotlib.cbook._warn_external(
-            'The left and right margins cannot be made large enough to '
-            'accommodate all axes decorations. ')
+        matplotlib.cbook._warn_external('The left and right margins cannot '
```

Comment:
just do `from matplotlib import cbook` at the top?

Reply:
Fixed, thanks.